lib/note-toolbar/index.tsx

```diff
@@ -1,21 +1,24 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
-import IconButton from '../icon-button';
 import BackIcon from '../icons/back';
+import IconButton from '../icon-button';
 import InfoIcon from '../icons/info';
+import NewNoteIcon from '../icons/new-note';
 import PreviewIcon from '../icons/preview';
 import PreviewStopIcon from '../icons/preview-stop';
 import RevisionsIcon from '../icons/revisions';
 import TrashIcon from '../icons/trash';
 import ShareIcon from '../icons/share';
 import SidebarIcon from '../icons/sidebar';
 import actions from '../state/actions';
+import { createNote } from '../state/ui/actions';
```

Comment:
note that this should already be available from `actions` as `actions.ui.createNote`

Reply:
Thank you! I updated based on this feedback.